.circleci/config.yml

```diff
@@ -48,7 +48,47 @@ jobs:
           command: |
             ./project.rb circle-deploy \
               --project all-of-us-workbench-test --creds-file circle-sa-key.json
-
+  api-build-integration-test:
+    docker:
+      - image: allofustest/workbench:buildimage-0.0.10
+    working_directory: ~/workbench
+    environment:
+      JVM_OPTS: -Xmx3200m
+      TERM: dumb
+    steps:
+      - checkout
+      - run:
+          name: Fetch Submodules
+          command: git submodule update --init
+      - restore_cache:
+          keys:
+          - api-cache-{{ checksum "~/workbench/api/build.gradle" }}
+          - api-cache-
+      - run:
+          name: Validate swagger definitions
+          working_directory: ~/workbench/api
+          command: ./project.rb validate-swagger --project-prop verboseTestLogging=yes
+      - run:
```

Comment:
Are all 3 of these SA keys needed for the integration test?

Reply:
Thinking about it, I think the only SA key we need is the circle one? And maybe the public API key, @danqrodney?

Reply:
I think you need the first step, in order to have permission to run gsutil in the third step.

public-api doesn't have integration tests so the second step isn't necessary probably.